Gruntfile.js

```diff
@@ -159,19 +235,50 @@ module.exports = function() {
 						}
 					]
 				}
+			},
+			"remove-sourcemappingurl": {
+				files: [
+					{
+						src: buildReleaseMin,
+						dest: buildReleaseMin
+					}
+				],
+				options: {
+					replacements: [
+						{
+							pattern: /\/\/# sourceMappingURL=.*/g,
+							replacement: ""
+						}
+					]
+				}
 			}
 		},
-		copy: {
-			// copy files to tests\build\boomerang-latest.js so tests/index.html points to the latest version always
+		strip_code: {
 			debug: {
+				files: [{
+					src: buildRelease
+				}],
+				options: {
+					start_comment: "BEGIN_DEBUG",
+					end_comment: "END_DEBUG"
+				}
+			},
+			prod: {
 				files: [
```

Comment:
Yes, because the production version removes debug code, logging etc.  OSS users may want to use dev and prod versions of the code.